sample-app/src/components/VisualAutocomplete/VisualSearchBar.tsx

```diff
@@ -94,6 +118,31 @@ export default function VisualSearchBar({
     />
   }
 
+  function renderRecentSearches() {
+    const options = recentSearches?.getRecentSearches().map(result => {
+      return {
+        value: result.query,
+        display: (
+          <div className='flex items-center'>
```

Comment:
would it be possible to use `renderAutocompleteResult` here?

Reply:
renderAutocompleteResult expects result to be of type AutocompleteResult and have a fix magnifying glass icon. I will try to make it more generic and see how it goes

Reply:
I forgot we were doing the whole renderHighlightedValue thing, so if it's annoying it's probably not worth it

Reply:
decided not to use it at the end